pkg/cmd/kube_bench.go

```diff
@@ -30,15 +37,25 @@ func NewKubeBenchCmd(cf *genericclioptions.ConfigFlags) *cobra.Command {
 			if err != nil {
 				return
 			}
-			report, node, err := kubebench.NewScanner(opts, kubernetesClientset).Scan(ctx)
+			starboardClientset, err := starboard.NewForConfig(config)
 			if err != nil {
 				return
 			}
-			starboardClientset, err := starboard.NewForConfig(config)
+			// List Nodes
+			nodeList, err := kubernetesClientset.CoreV1().Nodes().List(ctx, meta.ListOptions{})
 			if err != nil {
+				err = fmt.Errorf("list nodes: %w", err)
 				return
 			}
-			err = crd.NewWriter(ext.NewSystemClock(), starboardClientset).Write(ctx, report, node)
+			for _, nodeItem := range nodeList.Items {
+				klog.V(3).Infof("Node name: %s/%s", nodeItem.Name, nodeItem.Labels[masterNodeLabel])
+				report, node, err := kubebench.NewScanner(opts, kubernetesClientset).Scan(ctx, nodeItem.Name)
```

Comment:
The Scan() method is blocking and it takes some time to run the benchmarks. Instead of running the tests sequentially, I'm wondering if we can use goroutines and wait group to run tests in parallel, i.e. run a goroutine per cluster node.